EnglishFilter/sections/antiadblock.txt

```diff
@@ -544,6 +544,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/74115 
+deportealdia.live#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

Comment:
We have a similar rule. Exchange the rule by adding the domain to the similar rule.

Reply:
@zloyden Which rule?

Reply:
I added the domain the rules I knew that it should work but @Yuki2718 suggested to add this rule to consolidate (make it stronger).

Reply:
```
! https://github.com/AdguardTeam/AdguardFilters/issues/56588
key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

You should replace the task link and change the rule a bit.
```
! https://github.com/AdguardTeam/AdguardFilters/issues/74115
deportealdia.live,key-hub.eu#%#//scriptlet("set-constant", "adblockDetector.init", "noopFunc")
```

Reply:
Is it even needed? The large diffs disarms the anti-adblock

Reply:
Check.